migrations/versions/371a6bebcfb1_merge_feature_branch_and_main_branch_.py

```diff
@@ -0,0 +1,24 @@
+"""Merge feature branch and main branch migrations
```

Comment:
These 'merge' migration files are probably un-needed, right? I would suspect you can delete it, and then just make sure that the 'revision' / 'down_revision' is "chained together properly" for whatever files you end up with in the end